### ISSUE TITLE
[Fixed in GH-6052] Missing Visual Focus on Section5 Banner [Fix in #4294]

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#4298]
![Screen Shot 2019-08-28 at 10 32 08 PM](https://user-images.githubusercontent.com/15113219/63905882-b1d81b80-c9e3-11e9-9235-310b7ced1876.png)

Seems like the same issue as #4294 4294

**Comment by developpeurweb** [comment]
**For QA:**

- To test in Safari please refer to [these instructions](https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/5041#issuecomment-558883497).
- This got fixed in ticket GH-6052
- Theme `calydon` should be enabled to see changes.
- Visible focus outline overlapping adjacent ui elements may happen and may not be an issue, if in doubt please open a ticket per element or reach out to UI developer.
- Please separate visible focus issues (CSS: element without outline) from tabbing issues (HTML/TS: element not reachable with keyboard).

![Screen Shot 2020-01-31 at 4 42 20 PM](https://user-images.githubusercontent.com/15113219/73576770-3b838d80-4449-11ea-8443-ba0e9d0ce36c.png)

**Comment by KateChuen** [comment]
Tested on dev3.
- Chrome ✅
- Firefox ✅
- Safari ✅
